indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -635,20 +648,21 @@
                              {:name "all-collection-revisions"
                               :settings collection-setting-v1}]
                             :mapping collection-mapping}
-               ;; The granule configuration here initially only specifies a single collection indexes
-               ;; Additional granule indexes are created over time via the index set application.
                :granule {:indexes
-                         (cons {:name "small_collections"
-                                :settings granule-settings-for-small-collections-index}
-                               (for [idx extra-granule-indexes]
-                                 {:name idx
-                                  :settings granule-settings-for-individual-indexes}))
+                         (concat [{:name "small_collections"
+                                   :settings granule-settings-for-small-collections-index}
+                                  {:name "deleted-granules"
+                                   :settings deleted-granule-setting}]
+                                 (vec
+                                   (for [idx extra-granule-indexes]
+                                     {:name idx
+                                      :settings granule-settings-for-individual-indexes})))
 
                          ;; This specifies the settings for new granule indexes that contain data for a single collection
                          ;; This allows the index set application to know what settings to use when creating
                          ;; a new granule index.
                          :individual-index-settings granule-settings-for-individual-indexes
-                         :mapping granule-mapping}
+                         :mapping (merge granule-mapping deleted-granule-mapping)}
```

Comment:
The current configuration will cause both `granule` and `deleted-granule` be defined as types in all the granule related indices. I don't think we want that.